### PR TITLE
Add docs for LabelsOverrider/AnnotationsOverrider

### PR DESCRIPTION
**Pull request by chaunceyjiang**
Signed-off-by: chaunceyjiang <chaunceyjiang@gmail.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add docs for LabelsOverrider/AnnotationsOverrider
**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/karmada/issues/2641

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @RainbowMango

**Comment by RainbowMango** [comment]
Generally looks good to me. Please @Tingtal help to review it from a grammar point.

File: docs/userguide/scheduling/override-policy.md [hunk 1]
```diff
@@ -351,6 +353,60 @@ The allowed operations are as follows:
 
 Note: the usage of `ArgsOverrider` is similar to `CommandOverrider`, You can refer to the `CommandOverrider` examples.
 
+
+### LabelsOverrider
+
+The allowed operations are as follows:
+- `add`: the items in `value` will be appended to labels.
```
Comment (Tingtal) [inline docs/userguide/scheduling/override-policy.md:360]:
```suggestion
- `add`: The items in `value` will be appended to labels.
```

File: docs/userguide/scheduling/override-policy.md [hunk 2]
```diff
@@ -351,6 +353,60 @@ The allowed operations are as follows:
...
+- `remove`: if the item in `value` matches the item in labels, it will be deleted. If it does not match, nothing will be done.
```
Comment (Tingtal) [inline docs/userguide/scheduling/override-policy.md:361]:
```suggestion
- `remove`: If the item in `value` matches the item in labels, the former will be deleted. If they do not match, nothing will be done.
```

File: docs/userguide/scheduling/override-policy.md [hunk 3]
```diff
@@ -351,6 +353,60 @@ The allowed operations are as follows:
...
+- `replace`: if the key in `value` matches the key in the label, it will be replaced. If it does not match, it will not do anything
```
Comment (Tingtal) [inline docs/userguide/scheduling/override-policy.md:362]:
```suggestion
- `replace`: If the key in `value` matches the key in the label, the former will be replaced. If they do not match, nothing will be done.
```

File: docs/userguide/scheduling/override-policy.md [hunk 4]
```diff
@@ -351,6 +353,60 @@ The allowed operations are as follows:
...
+            value: 
+              foo: exist # "foo: foo" will be replaced by "foo: exist"
+          - operator: remove
+            value: 
+              baz: baz   # It will be removed from labels
+```
+
+### AnnotationsOverrider
+
+Note: the usage of `AnnotationsOverrider` is similar to `LabelsOverrider`, You can refer to the `LabelsOverrider` examples.
```
Comment (Tingtal) [inline docs/userguide/scheduling/override-policy.md:407]:
```suggestion
Note: `AnnotationsOverrider` functions the similar way as `LabelsOverrider`. You can refer to the `LabelsOverrider` examples.
```

**Review by Tingtal — CHANGES_REQUESTED** [review]
Language check.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**